fastlane/lib/fastlane/actions/danger.rb

```diff
@@ -13,12 +13,14 @@ def self.run(params)
         dangerfile = params[:dangerfile]
         base = params[:base]
         head = params[:head]
+        pr = params[:pr]
         cmd << "--danger_id=#{danger_id}" if danger_id
         cmd << "--dangerfile=#{dangerfile}" if dangerfile
         cmd << "--fail-on-errors=true" if params[:fail_on_errors]
         cmd << "--new-comment" if params[:new_comment]
         cmd << "--base=#{base}" if base
         cmd << "--head=#{head}" if head
+        cmd << "pr #{pr}" if pr
```

Comment:
Just to confirm, the parameter is `pr [id]` and not `--pr [id]`?

Reply:
It is in fact `pr [id]` and not `--pr [id]`. Version 3.5.2 changed it to this syntax. https://github.com/danger/danger/blob/master/CHANGELOG.md